packages/attention/src/component.tsx

```diff
@@ -185,13 +183,11 @@ export function Attention(props: AttentionProps) {
         {canClose && (
           <button
             type="button"
-            aria-label={i18n._(
-              /*i18n*/ {
```

Comment:
Is `/*i18n*/` something lingui's extractor doesn't need anymore to know where to look for things to extract?